.evergreen/run-tests-ubuntu.sh

```diff
@@ -116,12 +116,32 @@ else
     echo -e "SUCCESS: Apache site built and set for Kerberos auth\nActual Output:\n$CURL_OUTPUT"
 fi
 
+NODE_VERSION=${NODE_VERSION:-14}
+NODE_ARTIFACTS_PATH="${HOME}/node-artifacts"
+NPM_CACHE_DIR="${NODE_ARTIFACTS_PATH}/npm"
+NPM_TMP_DIR="${NODE_ARTIFACTS_PATH}/tmp"
+
+export NVM_DIR="${NODE_ARTIFACTS_PATH}/nvm"
+
+# create node artifacts path if needed
+mkdir -p ${NVM_DIR}
+mkdir -p ${NPM_CACHE_DIR}
+mkdir -p "${NPM_TMP_DIR}"
+
 echo "Installing Node.js"
 curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
```

Comment:
I notice the url is now `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh`
note the `nvm-sh` instead of `creationix`. Github is doing the redirect I suppose which is why this works but maybe we should update while we're here.